deploy/cdk.json

```diff
@@ -26,9 +24,34 @@
     "@aws-cdk/aws-ec2:uniqueImdsv2TemplateName": true,
     "@aws-cdk/core:checkSecretUsage": true,
     "@aws-cdk/aws-iam:minimizePolicies": true,
-    "@aws-cdk/core:target-partitions": [
-      "aws",
-      "aws-cn"
-    ]
+    "@aws-cdk/core:target-partitions": ["aws", "aws-cn"],
```

Comment:
OTOH, I think it's ok to leave this config UMCCR-specific, that way 3rd parties have a "working example" and it's also ergonomic for us to deploy from `main` if we wish to.